WalletWasabi.Fluent/ViewModels/Wallets/WalletViewModel.cs

```diff
@@ -55,13 +57,12 @@ protected WalletViewModel(Wallet wallet) : base(wallet)
 					.Select(_ => Unit.Default))
 				.Merge(Services.UiConfig.WhenAnyValue(x => x.PrivacyMode).Select(_ => Unit.Default))
 				.Merge(Wallet.Synchronizer.WhenAnyValue(x => x.UsdExchangeRate).Select(_ => Unit.Default))
+				.Merge(Settings.WhenAnyValue(x => x.MinAnonScoreTarget, x => x.MaxAnonScoreTarget).Select(_ => Unit.Default).Throttle(TimeSpan.FromMilliseconds(3000))
```

Comment:
```suggestion
				.Merge(Settings.WhenAnyValue(x => x.MinAnonScoreTarget, x => x.MaxAnonScoreTarget).Select(_ => Unit.Default).Skip(1).Throttle(TimeSpan.FromMilliseconds(3000))
```

Adding skip. So it will not fire on subscription. Without this, the whole homescreen would update when the wallet got loaded, and there would be another update after 3 seconds. Which is not necessary.